gcc/rust/hir/tree/rust-hir-item.h

```diff
@@ -1644,12 +1644,19 @@ class TupleStruct : public Struct
 class EnumItem : public Item
 {
   Identifier variant_name;
-
   Location locus;
 
 public:
   virtual ~EnumItem () {}
 
+  enum EnumItemKind
```

Comment:
This enum will be really nice so we can get rid of the separate enum-visitors in the type-checker